source/extensions/dynamic_modules/sdk/rust/Cargo.toml

```diff
@@ -8,8 +8,15 @@ license = "Apache-2.0"
 repository = "https://github.com/envoyproxy/envoy"
 
 [dependencies]
+mockall = "0.13.1"
 
 [build-dependencies]
 bindgen = "0.70.1"
 
 [lib]
+
+[features]
+# This feature is to enable mocking of Envoy objects for testing.
+# Since cfg(test) attribute is not propagated to dependencies, we need to define this feature
+# for downstream module developers to enable mocking in their tests.
+testing = []
```

Comment:
My 2c is that it's not worth it to have this feature since almost everyone will want to test, or at least make it a default feature, but not a big deal.